gemhardware/devices/src/common/amc/HwGenericAMC.cc

```diff
@@ -293,312 +218,249 @@ bool gem::hw::HwGenericAMC::linkCheck(uint8_t const& gtx, std::string const& opM
 }
 
 // FIXME UPDATE //
-// gem::hw::GEMHwDevice::OpticalLinkStatus gem::hw::HwGenericAMC::LinkStatus(uint8_t const& gtx)
+// gem::hw::GEMHwDevice::OpticalLinkStatus gem::hw::amc::HwGenericAMC::LinkStatus(uint8_t const& gtx)
```

Comment:
as long as everything's being updated, why not just remove this?
it's clearly not necessary for compatibility with any functionality

Reply:
basically it is a reference for future continued refactoring